packages/jaeger-ui/src/components/DeepDependencyGraph/getNodeLabel/index.tsx

```diff
@@ -0,0 +1,15 @@
+// Copyright (c) 2017 Uber Technologies, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+export { default } from './getNodeLabel';
```

Comment:
imo, I think this `index.tsx` file should just contain the contents of `getNodeLabel` instead of having this nearly empty file that just points to the real contents.
the go naming slides mention that file names should be path-aware, so `src/components/DeepDependencyGraph/getNodeLabel/getNodeLabel.tsx` would be redundant.

I followed this for `DeepDependecyGraph/index.tsx`, `DeepDependencyGraph/header.tsx`, and `DeepDependencyGraph/Graph.tsx` and it works nicely imo.
Might be worth removing the `Graph` part of the `DeepDependencyGraph` folder?

Reply:
I agree. Matching the URL seems to make sense: `DeepDependencies`